### ISSUE TITLE
Potential bug - users are not able to sign up for MapSwipe

### ISSUE THREAD

**Issue by nicolelaine** [mapswipe/python-mapswipe-workers#953]
An event happened in which 10% of participants were not able to sign up to MapSwipe. These users were across the web app and the mobile app. They received a message (on the web app, at least) that said "Sign-up failed."
<img width="442" alt="Screenshot 2024-07-17 at 13 08 59" src="https://github.com/user-attachments/assets/e006a9ff-f7f1-4651-9849-6e26bd8bd571">

We got some of the email addresses of the affected individuals. Many were at the same domain, but one was a hotmail address. None of the emails addressees existed in the MapSwipe system, so seems that the sign up did actually fail. 

More details to come.

**Waiting for info on:**

- Were the folks having trouble signing up having trouble on both versions (web and mobile), or were only both versions having trouble, as in folks had trouble signing up for either one, but not both?
- Was the error message the same on the phone?
- If they were on the web app, what browser they were using, and did they continue to have trouble on another browser?
- Was this a fully virtual event? Were folks all in the same place or on different networks?

**Comment by kopitek8** [comment]
@nicolelaine Any update on this?

**Comment by nicolelaine** [comment]
@kopitek8 Here is the further info I was able to obtain, and waiting for the rest..

- School students and teachers - all using the same type of address ( the school address)
- In multiple IT rooms, all on the same network.
- Inconsistent fault In as much that people in each room got in and people in each room didn’t 
- There was 140 folks and approx 40 didn’t get in… is there a sign up limit/time bound max sign up or something?

It does seem like most folks were using the school email address, but there was one person with a hotmail account that we know of that also could not sign up.

**Comment by Hagellach37** [comment]
Hey @nicolelaine ,
okay then this is a Firebase issue: New account creation --> 100 accounts/hour for each IP address

https://firebase.google.com/docs/auth/limits

**Comment by danbjoseph** [comment]
I am not sure we should remove this protection, but perhaps just add a warning into the event hosting information. I think the governance team should weigh-in if a change to the limit is considered.

**Comment by Hagellach37** [comment]
I agree. Also I read the docs that this would only be possible temporarily.
